src/ppx/ppx_eliom_utils.ml

```diff
@@ -53,6 +53,12 @@ let file_hash loc =
     let d = 10 - (i * 6) mod 8 in
     Bytes.set o i e.[(g p lsl 8 + g (p + 1)) lsr d land 63]
   done;
+  for i = 0 to 4 do
```

Comment:
Wasn't it possible to just remove the simple quote from input chars ?

Reply:
It would work with somewhat less-random names.